app/controllers/sessions_controller.rb

```diff
@@ -6,7 +6,7 @@ def new
 
   def create
     auth = request.env["omniauth.auth"]
-    user = User.where(:provider => auth['provider'], 
+    user = User.where(:provider => auth['provider'],
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.